metro.config.js

```diff
@@ -46,20 +46,14 @@ if (
   });
 } else {
   module.exports = {
-    serializer: {
-      getModulesRunBeforeMainModule: () => [
-        require.resolve('./Libraries/Core/InitializeCore'),
-      ],
-      getPolyfills,
-    },
     resolver: {
       platforms: ['ios', 'macos', 'android'],
       extraNodeModules: {
         'react-native': __dirname,
       },
     },
-    transformer: {
-      assetRegistryPath: require.resolve('./Libraries/Image/AssetRegistry'),
+    serializer: {
+      getPolyfills,
```

Comment:
@acoates-ms @tom-un These changes are likely wrong for RNM, but checking the history does not give me clues what these changes are needed for, do you recall? In the meantime I wanted to see if CI passes with these forked changes removed.

Reply:
So the CLI will set assetRegistryPath to 'react-native/Libraries/Image/AssetRegistry'.   And add a custom resolver that will then redirect react-native imports to react-native-macos when building macos packages.  

Does the RNBundle build with these settings?  If so, then its able to find RNM from within RNM, and so I think these settings should be fine.

We can nuke the whole first half of this file while here though - this config is no longer run from within the SDX repo anymore.  And that includes deleting the metro-resources.js file too.

Reply:
Thanks for the info 🙏 

The bundles build in the e2e tests. I’ll do some more testing to verify those, though.

Reply:
Had to revert this, it’s still needed when running from inside the RNM repo. I ended up making it conditional based on if it's being ran from inside the e2e tests.